### ISSUE TITLE
chore: Update `grafana` and `yarn` on the staging and production servers

### ISSUE THREAD

**Issue by GresilleSiffle** [tournesol-app/tournesol#398]
The command `sudo apt list --upgradable` list two upgrades available:
- `grafana`
- `yarn`

Note that the `grafana` minor version in production is behind staging.

**Comment by amatissart** [comment]
Note that these packages should be upgraded automatically during the deployment when this flag is set to `true` in the inventory:
https://github.com/tournesol-app/tournesol/blob/89f3d0a1a8abcccf50a0bac1d7b9e943ef6360c2/infra/ansible/inventory.yml#L121

**Comment by GresilleSiffle** [comment]
The packages have been automatically upgraded during the deployment on staging with:
```shell
./ansible/scripts/deploy-staging.sh apply notfast
```

I've also upgraded the production packages by setting the flag `upgrade_packages` to `true` in the Ansible inventory.

I don't remember why we choose to have a different flag for staging and production. Shouldn't the flags be both `false` or both `true`? Maybe we decided to not upgrate packages automatically?
